pkg/apps/apis/apps/validation/validation.go

```diff
@@ -160,8 +160,6 @@ func ValidateDeploymentConfigStatusUpdate(newConfig *appsapi.DeploymentConfig, o
 	statusPath := field.NewPath("status")
 	if newConfig.Status.LatestVersion < oldConfig.Status.LatestVersion {
 		allErrs = append(allErrs, field.Invalid(statusPath.Child("latestVersion"), newConfig.Status.LatestVersion, "latestVersion cannot be decremented"))
-	} else if newConfig.Status.LatestVersion > (oldConfig.Status.LatestVersion + 1) {
-		allErrs = append(allErrs, field.Invalid(statusPath.Child("latestVersion"), newConfig.Status.LatestVersion, "latestVersion can only be incremented by 1"))
```

Comment:
loosening the validation seems ok, if we won't start using it until it is present in the supported version skew 

@smarterclayton wdyt?